research_notebooks/explain_documentation/01_load_hummingbot_docs.ipynb

```diff
@@ -0,0 +1,728 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Loading Hummingbot\n",
+    "\n",
+    "We are going to use a Generic Loader and get the files from the scripts folder of the Hummingbot repository.\n",
+    "Also, we are using a LanguageParser to parse the files as Python code."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import time\n",
+    "\n",
+    "from dotenv import load_dotenv\n",
+    "from pprint import pprint\n",
+    "\n",
+    "load_dotenv()\n",
+    "\n",
+    "os.environ[\"OPENAI_API_KEY\"]\n",
+    "# you should see the api key if the .env file is loaded correctly"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "outputs": [],
+   "source": [
+    "from custom_loaders.mkdocs_site_loader import MkDocsSiteLoader\n",
+    "url = \"https://docs.hummingbot.org/\"\n",
```

Comment:
this should be `https://hummingbot.org/` in all places since we are no longer using docs sub-domain